quipucords/quipucords/settings.py

```diff
@@ -360,7 +360,7 @@ def app_secret_key_and_path():
 # Reverse default behavior to avoid host key checking
 os.environ.setdefault("ANSIBLE_HOST_KEY_CHECKING", "False")
 # Reverse default behavior for better readability in log files
-os.environ.setdefault("ANSIBLE_NOCOLOR", "False")
+os.environ.setdefault("ANSIBLE_NOCOLOR", "True")
```

Comment:
Should this change the preceding comment too?

Reply:
actually, the default behavior is having it set to False
When I added this I while ago I set it to False by mistake